Dockerfile

```diff
@@ -11,11 +11,7 @@ FROM registry.access.redhat.com/ubi8-minimal:8.1-398
 
 ARG KUBERNETES_VERSION=v1.17.3
 
-RUN case $(uname -m) in \
-       x86_64) ARCH="amd64" ;; \
-       ppc64le) ARCH="ppc64le" ;; \
-       s390x) ARCH="s390x";; \
-    esac && \
+RUN export ARCH="$(uname -m)" && if [[ ${ARCH} == "x86_64" ]]; then export ARCH="amd64"; fi && \
```

Comment:
hello, can we have a case for arm64 as well

```
# arch
aarch64
```
--> arm64 for download link
```
https://storage.googleapis.com/kubernetes-release/release/v1.17.3/bin/linux/arm64/kubectl
```

Reply:
@benoitf  - thanks , will raise a new PR to address this review comment .